integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -1017,76 +760,66 @@ private void createDomain() {
     p.setProperty("domain_logs", "/shared/logs");
     p.setProperty("production_mode_enabled", "true");
     assertDoesNotThrow(()
-        -> p.store(new FileOutputStream(domainPropertiesFile), "domain properties file"),
-        "Failed to write domain properties file");
+                    -> p.store(new FileOutputStream(domainPropertiesFile), "domain properties file"),
```

Comment:
Since multiple domains are going to exist with same t3 channel node port 32000, this might be a trouble.

Reply:
fixed